backend/src/services/v2/accessRequest.ts

```diff
@@ -61,6 +63,13 @@ export async function createAccessRequest(
     log.warn('Error when creating Release Review Requests. Approval cannot be given to this Access Request', error)
   }
 
+  sendWebhooks(
```

Comment:
If we're not awaiting, we should at least add a `.catch` to each of these with an error message.  Otherwise I think it'll get hidden?

Reply:
Rather than add error handling to every place this function is called, I added the error handling inside the function to avoid duplication. Happy to add more error handling inside the function if you think it's not sufficient.